tests/test_creation.py

```diff
@@ -146,7 +146,7 @@ def verify_files(root, config):
 
     existing_files = [f.relative_to(root) for f in root.glob("**/*") if f.is_file()]
 
-    assert sorted(existing_files) == sorted(expected_files)
+    assert sorted(existing_files) == sorted(set(expected_files))
```

Comment:
`pyproject.toml` gets added twice, since it's already there even if you don't choose it as a dependency file.